2001-3000/LIT2814RepCh98.xml

```diff
@@ -25,7 +29,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          </sourceDesc>
          <sourceDesc>
             <listWit>
-               <witness/>
+               <witness corresp="DSEthiop10"/>
```

Comment:
I think we should not add the manuscripts here, because we agreed to have as witness only manuscripts used for the edition we provide, whcih we don't, in this case.

Reply:
Ok